stanford_soe_r25.info

```diff
@@ -0,0 +1,10 @@
+name = R25 Room Reservations for Stanford SoE Intranet
+description = Interface for room reservations for 25Live/R25 for the Stanford SoE Intranet
+core = 7.x
+package = Stanford
+version = 7.x-1.0-alpha1
+project = stanford_soe_r25
+dependencies[] = stanford_r25
+stylesheets[all][] = css/stanford_soe_r25.css
```

Comment:
Is this stylesheet specific to a content type? If so, it might be nice to add the stylesheet conditionally on a preprocess hook that targets a content type rather than adding it to every page. This would also help you avoid having to write css selectors that start with `body.page-r25` as that is a really strong selector and makes it difficult for others to override later. 

eg: https://github.com/SU-SWS/stanford_story_page/blob/7.x-1.x/stanford_story_page.module#L25